bcipy/gui/BCInterface.py

```diff
@@ -369,7 +418,8 @@ def start_experiment(self) -> None:
         Using the inputs gathers, check for validity using the check_input method, then launch the experiment using a
             command to bci_main.py and subprocess.
         """
-        if self.check_input():
+
+        if self.check_input() and self.user_id_validator():
```

Comment:
This method should go in the `check_input` method, which already extracts the self.user.

Reply:
Oh that make sense! thank you for the feedback!